python/arcticdb/version_store/library.py

```diff
@@ -907,6 +907,35 @@ def read_metadata(self, symbol: str, as_of: Optional[AsOf] = None) -> VersionedI
         """
         return self._nvs.read_metadata(symbol, as_of)
 
+    def read_metadata_batch(self, symbol: str, as_of: Optional[AsOf] = None) -> VersionedItem:
```

Comment:
Since we're introducing a new public API, we should consider addressing that problem where multiple versions of the same sample cannot be queried in batch (which will be the most efficient use of the underlying API).

One possible solution is to return a `List[VersionedItem]` in the order of the arguments.

Reply:
I think the outcome of this was to use another API